src/lib/viewers/image/ImageViewer.js

```diff
@@ -390,8 +392,18 @@ class ImageViewer extends ImageBaseViewer {
         }
 
         if (this.controls && this.options.useReactControls) {
+            const canAnnotate =
+                this.areNewAnnotationsEnabled() &&
+                this.hasAnnotationCreatePermission() &&
+                this.currentRotationAngle === 0;
+
             this.controls.render(
                 <ImageControls
+                    annotationMode={this.annotationControlsFSM.getMode()}
+                    hasHighlight={false}
```

Comment:
Could we remove this property and set `false` directly on `AnnotationsControls` inside `ImageControls`?

Reply:
We can, but I figured keeping the two settings together was preferable. The `has` props default to false, anyway.